### PR TITLE
Removing others column

### PR DESCRIPTION
**Pull request by pdelewski**
This PR removes others as it is redundant to attributes (and maybe map in the future)

### CODE REVIEW

File: quesma/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -498,11 +489,8 @@ func (lm *LogManager) BuildInsertJson(tableName string, data types.JSON, inValid
 		return js, nil, nil
 	}
 	var attrsMap map[string][]interface{}
-	var othersMap SchemaMap
 	if len(config.attributes) > 0 {
-		attrsMap, othersMap, _ = BuildAttrsMapAndOthers(mDiff, config)
-	} else if config.hasOthers {
-		othersMap = mDiff
+		attrsMap, _ = BuildAttrsMapAndOthers(mDiff, config)
```
Comment (avelanarius) [inline quesma/clickhouse/clickhouse.go:493]: Nit: you could rename `BuildAttrsMapAndOthers` to `BuildAttrsMap`

File: quesma/clickhouse/clickhouse_test.go [hunk 2]
```diff
@@ -577,16 +565,16 @@ func TestJsonConvertingBoolToStringAttr(t *testing.T) {
...
 	assert.NoError(t, err)
-	assert.Equal(t, 0, len(others))
 	assert.Equal(t, 3, len(attrs))
 	for k := range attrs {
 		assert.Contains(t, k, "string")
 	}
 }
 
 func TestCreateTableString_1(t *testing.T) {
+	t.Skip("Others has been removed")
```
Comment (avelanarius) [inline quesma/clickhouse/clickhouse_test.go:577]: Maybe remove this test altogether then?

**Review by avelanarius — APPROVED** [review]
Looks OK, I suggested a couple more cleanups in my comments

File: quesma/clickhouse/clickhouse.go (hunk 1 above)
Comment (pdelewski) [reply quesma/clickhouse/clickhouse.go:493]: Right, thx

File: quesma/clickhouse/clickhouse_test.go (hunk 2 above)
Comment (pdelewski) [reply quesma/clickhouse/clickhouse_test.go:577]: You are right. I left it just to revisit this decision later

**Pull request merged**